go/worker/common/p2p/p2p.go

```diff
@@ -158,6 +160,14 @@ func (p *P2P) handleConnection(conn core.Conn) {
 	)
 }
 
+func (p *P2P) topicIDForRuntime(runtimeID common.Namespace) string {
+	return fmt.Sprintf("%s/%s/%s",
```

Comment:
BTW, is there a limit on topic length?

Reply:
Doesn't seem like it. The type is `string` and I don't see any checks for length through the code, also the [specs](https://github.com/libp2p/specs/tree/master/pubsub#the-topic-descriptor) don't specify any such limit.

Reply:
Ok, just making sure we don't run into any limits :)